analysis/configs/tsneLSF.yaml

```diff
@@ -7,7 +7,7 @@ model_name: 'LSF'
 
 file_prefix: "LSF-test-set"
 img_res: 256
-dataset_path: "/fastscratch/elhamod/data/Fish/taming_transforms_fish_test_padded_256_imagenetmeancolor.txt"
+dataset_path: "/fastscratch/elhamod/data/Fish/fish_test.txt"
 
 ckpt_path: /home/harishbabu/projects/taming-transformers/logs/2023-02-01T23-38-48_LSF2-VQVAE1024emb-256img_imagenetmean_noaug-base_noaug_withimgnet/checkpoints/last.ckpt
 yaml_path: /home/harishbabu/projects/taming-transformers/logs/2023-02-01T23-38-48_LSF2-VQVAE1024emb-256img_imagenetmean_noaug-base_noaug_withimgnet/configs/2023-02-01T23-38-48-project.yaml
```

Comment:
rename the models to be userfriendly and upload the same name to google drive. take a look at my yamls for examples

Reply:
Updated the model paths. Should we also change the absolute paths to relative paths like
"/home/harishbabu/projects/phylonn/logs/LSF/checkpoints/last.ckpt" to "./logs/LSF/checkpoints/last.ckpt"